table.css.scss

```diff
@@ -0,0 +1,10 @@
+@import 'table.scss';
+
+.vui-table {
+	@include vui-table;
+}
+
+.vui-table-header th {
```

Comment:
I'm confused as to why you need this rule. Isn't the `vui-table` mixin already going to apply header styles to `thead > th` elements?